cve_bin_tool/cli.py

```diff
@@ -55,6 +58,11 @@ class Scanner(object):
     def __init__(self, cvedb, checkers=None, logger=None):
         if logger is None:
             logger = LOGGER.getChild(self.__class__.__name__)
+        # Update egg if installed in development mode
+        if not IS_PRODUCTION():
```

Comment:
This is how you can import a specific file (prepending to sys.path and using importlib can sometimes result in the wrong file being imported if there's another file in the current working directory with the same name, although if you have a way around this I'd be stoked, just what I've observed).

```python
import os
import importlib.util
from setuptools import setup

egg_updater_path = os.path.join(os.path.dirname(__file__), "..", "egg_updater.py")
if os.path.isfile(egg_updater_path):
    spec = importlib.util.spec_from_file_location(
        "egg_updater", 
    )
    egg_updater = importlib.util.module_from_spec(spec)
    spec.loader.exec_module(egg_updater)
else:
    egg_updater = None
```

Then

```suggestion
        if egg_updater is not None:
```